modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/webpack_federation.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+Liferay = window.Liferay || {};
+
+Liferay.Webpack = {
```

Comment:
Did we decide to just leave `Liferay.Webpack`? I know it's unlikely we can ever use whatever is in there if we ever change our implementation detail, but it still feels way too obvious.

Why not:
- `Liferay.__MODULES__`?
- `Liferay.__FEDERATION__`?
- `Liferay.__SHARED__`?
- ...

Reply:
Decide is not the word, but I wanted to unblock this so I went with what I commented [here](https://github.com/liferay-frontend/liferay-portal/pull/527#discussion_r530510140).

I simply wanted to put all the webpack stuff under one single place to avoid polluting the `Liferay` namespace more, but it is true that if we want to put something else related to webpack, in the future, there, it's going to be a bit ugly.

I really don't find too much difference between different solutions. However, knowing that:

1. `SharedScope` is fully internal and can be moved to any other place we like in the future.
2. `Container` is internal too, but moving it to another place would require all projects to be rebuilt :grimacing: .

may help with the decision...

Reply:
Let's push if green and we can change afterwards. Let's just make sure we have a final decision before we our first release.

Reply:
Makes sense :+1: . Until we see the final solution we don't have the whole picture.